src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -956,6 +957,115 @@ TEST_F(MessageHelperTest, SubscribeApplicationToSoftButton_CallFromApp) {
       message_params, appSharedPtr, function_id);
 }
 
+TEST(MessageHelperTestCreate,
+   CreateHashUpdateNotification_FunctionId_Equal) {
+  uint32_t app_id = 0;
+  smart_objects::SmartObjectSPtr ptr =
+      MessageHelper::CreateHashUpdateNotification(app_id);
+
+  EXPECT_TRUE(ptr);
+
+  smart_objects::SmartObject& obj = *ptr;
+
+  int function_id =
+      static_cast<int>(mobile_apis::FunctionID::OnHashChangeID);
+  int notification = static_cast<int>(::application_manager::kNotification);
+
+  EXPECT_EQ(function_id, obj[strings::params][strings::function_id].asInt());
+  EXPECT_EQ(app_id, obj[strings::params][strings::connection_key].asUInt());
+  EXPECT_EQ(notification,
```

Comment:
@Kozoriz, so, then, at least make this variables const, if you don't change them.